.github/workflows/node.js.yml

```diff
@@ -0,0 +1,27 @@
+name: Node.js CI
```

Comment:
I've enjoyed learning a bit more about GitHub's CI solution, GitHub Actions. This file should include under steps `run: npm test` but currently there are no tests to run.